lib/protobuf/message.rb

```diff
@@ -238,3 +240,5 @@ def copy_to(object, method)
 
   end
 end
+
+require 'protobuf/descriptors'
```

Comment:
No, get_option needs Google::Protobuf::Options to work (Optionable takes
a Google::Protobuf::Options class as an argument), so descriptor.pb
needs to get required before using get_option.

BUT, to parse descriptor.pb, Protobuf::{Enum,Message,etc} needs to
exist. So, require protobuf/message, then require descriptor.pb

I suppose I could add a comment lol